src/modules/emotes/socket-client-listeners.js

```diff
@@ -0,0 +1,55 @@
+import socketClient, {deserializeSocketChannel, EventNames} from '../../socket-client.js';
+import {getCurrentChannel} from '../../utils/channel.js';
+import twitch from '../../utils/twitch.js';
+import watcher from '../../watcher.js';
+import channelEmotes from './channel-emotes.js';
+
+function validChannelDestination(channel) {
+  const currentChannel = getCurrentChannel();
+  const [provider, providerId] = deserializeSocketChannel(channel);
+  return currentChannel.id === providerId && currentChannel.provider === provider;
+}
+
+socketClient.on(EventNames.EMOTE_CREATE, ({channel, emote}) => {
+  if (!validChannelDestination(channel)) {
+    return;
+  }
+
+  channelEmotes.upsertChannelEmote(emote);
+
+  watcher.emit('emotes.updated');
+  twitch.sendChatAdminMessage(`${emote.code} has been added`, true);
```

Comment:
nit: `BetterTTV: The emote ${emote.code} has been added`

Reply:
what about `BetterTTV: ${emote.code} has been added` ?